dist/bump-release.sh

```diff
@@ -13,8 +13,8 @@ VERSIONTAG="v${VERSION}"
 TAGBR="v${VERSION}-tag"
 
 replace_version() {
```

Comment:
I don't think we need this function at all now. There's no point in creating these bump commits any more. We can just tag the release and that's it.